flaky-tests/ci-providers/gitlab.md

```diff
@@ -56,14 +59,14 @@ unit-test-job:   # This job runs the tests
 upload_test_results: # This job uploads tests results run in the last stage
   stage: flaky-tests
   script:
-    - curl -fsSL --retry 3 "UPLOADER_LINK" -o ./trunk-analytics-uploader
-    - chmod +x ./trunk-analytics-uploader
-    - ./trunk-analytics-uploader upload --junit-paths "tests/jest/jest_junit_test.xml" --org-url-slug <TRUNK_ORG_SLUG> --token $TRUNK_API_TOKEN
+    - curl -fsSL --retry 3 "CLI_LINK" | tar -xvf > ./trunk-analytics-cli
+    - chmod +x ./trunk-analytics-cli
+    - ./trunk-analytics-cli upload --junit-paths "tests/jest/jest_junit_test.xml" --org-url-slug <TRUNK_ORG_SLUG> --token $TRUNK_API_TOKEN
```

Comment:
an aside: Anyone can test this out locally by trying to run `./trunk-analytics-cli --version` 😄